pkg/controllers/user/rbac/namespace_handler.go

```diff
@@ -168,6 +194,28 @@ func (n *nsLifecycle) reconcileNamespaceProjectClusterRole(ns *v1.Namespace) err
 					r.ResourceNames = resNames
 				}
 			}
+			//if ResourceNames is empty, delete the rule and delete the role if no rules exist
+			toDeleteRules := 0
+			for _, rule := range undesiredRole.Rules {
+				if len(rule.ResourceNames) == 0 {
+					toDeleteRules++
+				}
+			}
+			if toDeleteRules == len(undesiredRole.Rules) {
+				logrus.Debugf("Deleting ClusterRole %s", undesiredRole.Name)
```

Comment:
Could you actually bump this and all logs youve added where you are about to delete something up to INFO?

I wont to go back through all my code and log at info whenever i am actually creeating, updating, dleeting seomthing